tools/bazel.rc

```diff
@@ -40,6 +40,7 @@ build:tsan --copt=-DGRPC_TSAN
 build:tsan --copt=-DTHREAD_SANITIZER  # used by absl
 build:tsan --linkopt=-fsanitize=thread
 build:tsan --action_env=TSAN_OPTIONS=suppressions=test/core/util/tsan_suppressions.txt:halt_on_error=1:second_deadlock_stack=1
+build:tsan --test_env=TSAN_OPTIONS=report_atomic_races=0
```

Comment:
this looks like it would override the `--action_env=TSAN_OPTIONS=` settings on the previous line. I'd suggest just appending `:report_atomic_races=0` to the previous line.